docs/docs/general/03-Element Selection/custom-elements.md

```diff
@@ -41,7 +41,7 @@ Arguments:
 - **customImage** (*`string`, required*):
     - A cropped image in the form of a base64 string or file path.
 - **name** (*`string`, optional*):
-    - A unique name that can be used for filtering for the custom element. If not given, any text inside the custom image will be detected via OCR. You can use [get()](../../api/06-Getters/get.md) to extract text from images or logos because of this. TODO test this!
+    - A unique name that can be used for filtering for the custom element. If not given, any text inside the custom image will be detected via OCR. You can use [get()](../../api/06-Getters/get.md) to extract text from images or logos because of this.
```

Comment:
Let's remove the OCR part for now as it does not work so we don't set the wrong expectation or add a hint that there is a known issue at the moment.